Panic with 0.5 (racy use of timers)

#### Version information:
go-ipfs version: 0.5.0-dev-67741060c
Repo version: 7
System version: amd64/linux
Golang version: go1.14

#### Description:
Basically pulled the source and just built it and ran in daemon mode.  Nothing in particular triggers this as it happens on it's own. The only thing I did was add this node as a peer from another non-0.5 node (0.4.22). Here is the stack trace:
```
panic: runtime error: racy use of timers

goroutine 1130 [running]:
time.resetTimer(0xc0006fa5a8, 0x78e3267cb52c)
        /usr/local/go/src/runtime/time.go:233 +0x35
time.(*Timer).Reset(0xc0006fa5a0, 0x6fc23ac00, 0xc000208b10)
        /usr/local/go/src/time/sleep.go:126 +0x7d
github.com/libp2p/go-yamux.(*Session).recvLoop(0xc0007bf110, 0x0, 0x0)
        pkg/mod/github.com/libp2p/go-yamux@v1.3.1/session.go:516 +0x302
github.com/libp2p/go-yamux.(*Session).recv(0xc0007bf110)
        pkg/mod/github.com/libp2p/go-yamux@v1.3.1/session.go:483 +0x2b
created by github.com/libp2p/go-yamux.newSession
        pkg/mod/github.com/libp2p/go-yamux@v1.3.1/session.go:129 +0x379
```

--- a/go.mod
+++ b/go.mod
@@ -80,7 +80,7 @@ require (
 	github.com/libp2p/go-libp2p-swarm v0.2.2
 	github.com/libp2p/go-libp2p-testing v0.1.1
 	github.com/libp2p/go-libp2p-tls v0.1.3
-	github.com/libp2p/go-libp2p-yamux v0.2.3
+	github.com/libp2p/go-libp2p-yamux v0.2.4
 	github.com/libp2p/go-maddr-filter v0.0.5
 	github.com/libp2p/go-socket-activation v0.0.2
 	github.com/mattn/go-runewidth v0.0.8 // indirect

